test/studies/amr/lib/cfboundary/RefiningTransfer.chpl

```diff
@@ -21,8 +21,8 @@ class GridCFGhostRegion {
   //|/...............|/
 
   const grid:             Grid;
-  const coarse_neighbors: domain(Grid);
-  const transfer_regions:  [coarse_neighbors] MultiDomain(dimension,stridable=true);
+  var coarse_neighbors: domain(Grid);
+  var transfer_regions:  [coarse_neighbors] MultiDomain(dimension,stridable=true);
```

Comment:
If you wanted to leave these const, you could make a local variable that gets updated during the computation, and then copy that variable into the const field when you were done building it (though that would obviously have the memory impact creating a copy entails).